tests/unit/ConsentContinuationOut.spec.ts

```diff
@@ -121,34 +124,47 @@ describe('Consent to Continuation Out view', () => {
     vm.courtOrderValid = true
     vm.detailCommentValid = true
     vm.documentDeliveryValid = true
+    vm.foreignJurisdictionValid = true
     expect(!!vm.isPageValid).toBe(true)
 
     // verify "validated" - invalid Detail Comment form
     vm.certifyFormValid = true
     vm.courtOrderValid = true
     vm.detailCommentValid = false
     vm.documentDeliveryValid = true
+    vm.foreignJurisdictionValid = true
     expect(!!vm.isPageValid).toBe(false)
 
     // verify "validated" - invalid Certify form
     vm.certifyFormValid = false
     vm.courtOrderValid = true
     vm.detailCommentValid = true
     vm.documentDeliveryValid = true
+    vm.foreignJurisdictionValid = true
     expect(!!vm.isPageValid).toBe(false)
 
     // verify "validated" - invalid Court Order form
     vm.certifyFormValid = true
     vm.courtOrderValid = false
     vm.detailCommentValid = true
     vm.documentDeliveryValid = true
+    vm.foreignJurisdictionValid = true
     expect(!!vm.isPageValid).toBe(false)
 
     // verify "validated" - invalid Document Delivery form
     vm.certifyFormValid = true
     vm.courtOrderValid = true
     vm.detailCommentValid = true
     vm.documentDeliveryValid = false
+    vm.foreignJurisdictionValid = true
+    expect(!!vm.isPageValid).toBe(false)
+
+    // verify "validated" - invalid Foreign Jurisdiction form
+    vm.certifyFormValid = true
+    vm.courtOrderValid = true
+    vm.detailCommentValid = true
+    vm.documentDeliveryValid = true
+    vm.foreignJurisdictionValid = false
     expect(!!vm.isPageValid).toBe(false)
```

Comment:
Why do you need `!!` ? Isn't "isPageValid" already a boolean?

Reply:
I see the other tests above do the same thing. Can it be updated for all of them?